Lombiq.Tests.UI/Extensions/ScriptingUITestContextExtensions.cs

```diff
@@ -1,16 +1,16 @@
+using Atata;
 using Lombiq.Tests.UI.Services;
 using Newtonsoft.Json;
-using OpenQA.Selenium;
 
 namespace Lombiq.Tests.UI.Extensions;
 
 public static class ScriptingUITestContextExtensions
 {
     public static object ExecuteScript(this UITestContext context, string script, params object[] args) =>
-        context.ExecuteLogged(nameof(ExecuteScript), script, () => ((IJavaScriptExecutor)context.Driver).ExecuteScript(script, args));
+        context.ExecuteLogged(nameof(ExecuteScript), script, () => context.Driver.AsScriptExecutor().ExecuteScript(script, args));
 
     public static object ExecuteAsyncScript(this UITestContext context, string script, params object[] args) =>
-        context.ExecuteLogged(nameof(ExecuteAsyncScript), script, () => ((IJavaScriptExecutor)context.Driver).ExecuteAsyncScript(script, args));
+        context.ExecuteLogged(nameof(ExecuteAsyncScript), script, () => context.Driver.AsScriptExecutor().ExecuteAsyncScript(script, args));
```

Comment:
I now see that we have several of these `AsScriptExecutor()`s so an extension would be nice still.

Reply:
We only have 3 and out of that 2 are in this file for the context extensions, so I don't think we need it.